pandas/io/pytables.py

```diff
@@ -616,7 +616,7 @@ def keys(self, include: str = "pandas") -> List[str]:
 
         include : str, default 'pandas'
                 When kind equals 'pandas' return pandas objects
-                When kind equals 'native' return native HDF5 Table objects
+                When kind equals 'native' return native HDF5 Table objects.
```

Comment:
The rendering of this can be seen here https://pandas.pydata.org/pandas-docs/dev/reference/api/pandas.HDFStore.keys.html?highlight=keys#pandas.HDFStore.keys

looks like the previous line requires a period too.

Reply:
Ah yes, ok